### ISSUE TITLE
error querying certificate length: input/output error

### ISSUE THREAD

**Issue by aep** [google/go-sev-guest#18]
when using the structure from https://github.com/AMDESE/linux-svsm

i'm setting the certs using https://github.com/AMDESE/sev-guest

as hinted in

https://github.com/AMDESE/linux-svsm/issues/14

which works fine with sev-guest-get-report but not with this lib.

pressumably svsm doesnt implement the right call? 
can we just assume a default size if it fails?

**Comment by aep** [comment]
never mind, this is unrelated to svsm. it's also broken without it. possibly the way sev-guest sets the certs is incompatible with go-sev-guest?

**Comment by aep** [comment]
ignoring the error here makes it work

https://github.com/google/go-sev-guest/blob/bde9adb9a9e1a56a4a77d6929e8a1b3df685683b/client/client.go#L103

not sure what the correct fix is

**Comment by aep** [comment]
oh you fixed it already. :D


can confirm 2790cd88bf58682313e70604fe35a04340c12d28 solves it for me, thanks!

**Comment by aep** [comment]
err oops, that isnt merged yet

**Comment by deeglaze** [comment]
Yes, give me a moment to mess with some real hardware. We didn't have a flow set up to test with a non-empty certificate buffer for reasons. I'll manually try it out though.
